modules/claims_api/spec/requests/v2/veterans/disability_compensation_request_spec.rb

```diff
@@ -325,7 +325,7 @@
         context 'when the cert is false' do
           let(:claimant_certification) { false }
 
-          it 'responds with a 200' do
+          it 'responds with a bad request' do
```

Comment:
This line was just not describing the outcome for the test correctly (:unprocessable_entity) so I just I just refactored it here since it was such a small change